lti_consumer/lti_xblock.py

```diff
@@ -1507,6 +1507,18 @@ def get_lti_1p3_custom_parameters(self):
         """
         custom_parameters = {}
 
+        # Get site-wide extra parameters from processor functions.
+        for processor in self.get_parameter_processors():
+            try:
+                default_params = getattr(processor, 'lti_xblock_default_params', {})
+                custom_parameters.update(default_params)
+                custom_parameters.update(processor(self) or {})
+            except Exception:  # pylint: disable=broad-except
```

Comment:
I think it's better to be more especific with the exception. What do you think about that?

Reply:
What kind of exceptions you think we should catch?, I think a processor can raise any type of exception